src/core/public/chrome/ui/header/header_logo.tsx

```diff
@@ -96,32 +98,45 @@ function onClick(
   }
 }
 
-export const DEFAULT_DARK_LOGO = 'opensearch_logo_dark_mode.svg';
-export const DEFAULT_LOGO = 'opensearch_logo_default_mode.svg';
+export const DEFAULT_LOGO = 'plugins/home/assets/logos/opensearch_dashboards.svg';
+export const DEFAULT_DARK_LOGO = 'plugins/home/assets/logos/opensearch_dashboards_darkmode.svg';
```

Comment:
This seems backwards to me - why is a core service dependent on a plugin path? I'd expect all core services to be independent of removing any plugin.

Reply:
I agree. I should prehaps look for a better place to put this assets.